core/src/main/java/novoda/lib/sqliteprovider/util/DBUtils.java

```diff
@@ -125,31 +131,38 @@ public static String getSQLiteVersion() {
     @Deprecated
     public static List<String> getUniqueConstrains(SQLiteDatabase db, String table) {
         List<String> constrains = new ArrayList<String>();
-        final Cursor pragmas = db.rawQuery(String.format(PRGAMA_INDEX_LIST, table), null);
-        while (pragmas.moveToNext()) {
-            int isUnique = pragmas.getInt(2);
+        final Cursor indices = queryIndexListForTable(table, db);
```

Comment:
It's deprecated... so, no? If someone updated the lib to get the fix, they could switch to using the non-deprecated version.